agent-lambda/build.gradle

```diff
@@ -18,17 +18,13 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
```

Comment:
Is above import still needed?

Reply:
nope

Reply:
Oops, commented too soon! It'll be needed for next pass 😁